template.yaml

```diff
@@ -27,6 +27,14 @@ Parameters:
     Type: Number
     Default: 86400
     Description: Maximum number of seconds until cached responses expire; default 86400 (1 day)
+  CodesToOmit:
+    Type: String
+    Description: Comma-delimited list of program codes to ignore (treat as inactive).
```

Comment:
yeah, this is taking in a list of numeric codes not a list of valid tag strings because I didn't want to have to include the program name in this list

Reply:
I'm not using the terms interchangeably, `program codes` are strictly numeric and `tags` are strings that contain friendly names. Would it make more sense if I renamed `program codes` to `mip codes`?

Reply:
i still don't know what `mips` means.  how about just `costcenters` or `programs`?

Reply:
I'm not quite sure where I got the s in `mips`, maybe from Bruce's script, but MIP (aka MIP Cloud) is the third-party api we are querying. I don't like `costcenters` vs `programs` because I want the name to indicate which one is strictly numeric and which one is a friendly name, so I've been trying to keep `code` in the name for the numeric code.

Reply:
ok, i think there's a few places where it's described as `mips` including the architecture diagram.